system/lib/libcxx/include/__config

```diff
@@ -40,8 +40,14 @@
 #define _LIBCPP_VERSION 13000
 
 #ifndef _LIBCPP_ABI_VERSION
+#if defined(__EMSCRIPTEN__)
```

Comment:
I think this should go in `__config_site` rather than in `__config`

Reply:
Done